datasets/glossis/glossis.py

```diff
@@ -144,7 +147,10 @@
             logging.info(f"Added task {taskid}")
             taskids.append(taskid)
 
-    if not args.skip_currents:
+        # Wait for all the tasks to finish
+        wait_gee_tasks(taskids)
```

Comment:
Why do we start waiting at each dataset? We could go on with calculations in the same script and wait for everyting at the end of this if/else statement?

Reply:
Because each dataset runs in it's own pod in parallel.